lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -87,9 +88,9 @@ class ProgressTabView(RetrieveAPIView):
                          'Pass' is not included.
         studio_url: (str) a str of the link to the grading in studio for the course
         verification_data: an object containing
-            link: (str) the link to either start or retry verification
-            status: (str) the status of the verification
-            status_date: (str) the date time string of when the verification status was set
+            link: (str) the link to either start or retry ID verification
+            status: (str) the status of the ID verification
+            status_date: (str) the date time string of when the ID verification status was set
```

Comment:
My attempt to clear up confusion. `verification_data` corresponds to IDV, `verified_mode` corresponds to audit/verified

Reply:
Good thinking and I appreciate that! Thanks Carla!